NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Ticket 37- Post Image

Description
Added ability to upload post images. [Trello Ticket #37](https://trello.com/c/0c0rvFJm)

Type of Change
-Added new, non-breaking feature.

Testing Instructions


1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout sf-ticket-37
5. If you don't see the most recent version of my files, git pull origin sf-ticket-37
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in you client folder. Run "npm start" in your terminal.
11. Login as an admin. [foo@bar.com](mailto:foo@bar.com) for the email parameter. Leave the password blank. Click "Login."
12. Click "My Posts"
13. Click on the title of a post to see the details page.
14. Click upload an image, and then submit.
15. The post image should be updated.
16. 

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
